types.proto

```diff
@@ -39,3 +38,11 @@ message AddressWithBlindingKey {
   // The blinding private key for the given address encoded in hex format
   string blinding = 2;
 }
+
+message FeeInfo {
+  string trade_id = 1;
+  int64 fee_basis_point = 2;
```

Comment:
we can skip `fee_` prefix since is self ecident from the message name

Reply:
Also move it into operator.proto, since we do not use accross files better like this (ie. trader facing sdk do not import it)